integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Kubernetes.java

```diff
@@ -504,18 +504,18 @@ public static V1Pod getPod(String namespace, String labelSelector, String podNam
    * @param serviceName name of the service to check for
    * @param label the key value pair with which the service is decorated with
    * @param namespace the namespace in which to check for the service
-   * @return true if the service is found otherwise false
-   * @throws ApiException when there is error in querying the cluster
+   * @return true if the service is found, otherwise false. If there is
+   *              an error in querying the cluster, it returns false.
    */
-  public static boolean doesServiceExist(
-      String serviceName, Map<String, String> label, String namespace)
-      throws ApiException {
-    boolean exist = false;
-    V1Service service = getService(serviceName, label, namespace);
-    if (service != null) {
-      exist = true;
+  public static boolean doesServiceExist(String serviceName, Map<String, String> label, String namespace) {
+    try {
```

Comment:
It doesn't seem right to suppress an exception since then we wouldn't see it in the retries. The implementation of getService() appears to be able to return the instance or null in normal cases, so wouldn't an ApiException represent a real problem? (And, yes, it should retry if an attempt had an exception, correct?)

Reply:
Currently the test fails if there's an ApiException and doesn't seem to do retries. See - https://build.weblogick8s.org:8443/job/wko-kind-nightly-parallel/861/testReport/oracle.weblogic.kubernetes/ItElasticLoggingFluentd/___/ . Do the integration tests have retry mechanism similar to how it's done in the operator?

My understanding is that with my change, if there's an APIException, the method `doesServiceExist` will return `false` and the check will be performed again until the service exists.

Reply:
https://github.com/awaitility/awaitility/wiki/Usage#ignoring-exceptions

Options: 
* Use Awaitility's ignoreExceptions(), or
* Swallow the exception, but I'd prefer it inside the lambda of the testUntil() rather than in the general-purpose method. 

Also, I can't figure out why the lamba has assertDoesNotThrow() rather than just catching the exception and having the conditional evaluate to false.